horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/RunServiceImpl.java

```diff
@@ -922,10 +922,14 @@ private void trashConnectedDatasets(int runId, int testId) {
    @Override
    public void updateDescription(int id, String description) {
       // FIXME: fetchival stringifies the body into JSON string :-/
+      if(description.isEmpty())
+         throw ServiceException.badRequest("Description can not be empty");
+
```

Comment:
An empty description is a valid state. How do we handle the case if someone erroneous changes the description and then wants to remove that description?

Reply:
No objections :)

Reply:
My point was if someone wants to remove the description, this will throw a `ServiceException.badRequest` exception. The endpoint will not behave as expected

Reply:
i see you have taken this out now :)